dts/bindings/pwm/renesas,pwm-rcar.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022, IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas R-Car PWM controller
+
+compatible: "renesas,pwm-rcar"
+
+include: [pwm-controller.yaml, pinctrl-device.yaml, base.yaml]
+
+properties:
+    "#pwm-cells":
+      const: 2
+
+pwm-cells:
+  - channel
+  - flags
```

Comment:
```suggestion
pwm-cells:
  - channel
# period in terms of nanoseconds
  - period
  - flags
```

This is the standard convention.

Reply:
Hi Henrik, thanks. I didn't saw this conventional. I will update it based on the #45013 PR.

Reply:
Ok, Done ! I did the update with a rebase on main. Do not hesitate to tell me if you see anything else. Thank you @henrikbrixandersen